imports/api/genes/hectar/hectar.test.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-env mocha */
+import { resetDatabase } from 'meteor/xolvio:cleaner';
+import chai from 'chai';
+import logger from '../../util/logger';
+import { hectarCollection } from './hectarCollection';
+import addHectar from './addHectar';
+import { addTestUsers, addTestGenome } from '../../../startup/server/fixtures/addTestData';
+import '../../jobqueue/process-hectar';
+
+describe('hectar', function testHectar() {
+  let adminId;
+  let newUserId;
+  let adminContext;
+  let userContext;
+
+  logger.log('Testing Hectar methods');
+
+  beforeEach(() => {
+    ({ adminId, newUserId } = addTestUsers());
+    adminContext = { userId: adminId };
+    userContext = { userId: newUserId };
+  });
+
+  afterEach(() => {
+    resetDatabase();
+  });
+
+  it('Should add Hectar tab file', function importhectar() {
+    // Increase timeout
+    this.timeout(20000);
+
+    addTestGenome(annot = true);
+
+    const hectarParams = {
+      fileName: 'assets/app/data/Bnigra_hectar.tsv',
+    };
+
+    // Should fail for non-logged in
+    chai.expect(() => {
+      addHectar._execute({}, hectarParams);
+    }).to.throw('[not-authorized]');
+
+    // Should fail for non admin user
+    chai.expect(() => {
+      addHectar._execute(userContext, hectarParams);
+    }).to.throw('[not-authorized]');
+
+    const { result } = addHectar._execute(adminContext, hectarParams);
+
+    chai.assert.equal(result.nInserted, 1)
+
+    const hecs = hectarCollection.find({ protein_id: 'BniB01g000010.2N.1-P' }).fetch();
```

Comment:
Protein_id in the test file is "BniB01g000010.2N.1", so the query return no results